src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java

```diff
@@ -41,7 +43,7 @@ public Boolean isServerUp() {
             return false;
         } catch (StatusCodeError error) {
             CodeSyncLogger.error(
-                String.format("Server returned %s status code on health check endpoint.", error.getStatusCode())
+                    String.format("Server returned %s status code on health check endpoint.", error.getStatusCode())
```

Comment:
I have been using 3 space indent for almost all files in the code. Can you switch to that as well?